config/crd/bases/ramendr.openshift.io_drpolicies.yaml

```diff
@@ -150,6 +150,9 @@ spec:
             required:
             - drClusters
             type: object
+            x-kubernetes-validations:
+            - message: Value is immutable
+              rule: self == oldSelf
```

Comment:
Looks like we add the same information twice - once in the kubebuilder comment, and once here. Can we have this only once? What happens if the rule and message are different in both places?

Also this looks really magical - can you link to the docs describing this special syntax?